### DISCUSSION TITLE
Agenda Meeting 12.12.24

### DISCUSSION THREAD

**Discussion by jac0mp** [AuDojo/AuDojo#125] [Agenden]
# Agenda

### Frontend

- Hauptseite: Trang & An
   - PivotElement markiert
   - Besserer SortGuide
   - ToolTips für falschen Input und ShortCuts
   - 
 
- Tutorial: Minseo
- Impressum & Datenschutz: Jan

- Fragen:
  - Wie können wir dafür sorgen das neuer Frontendzuwachs möglichst effizient eingearbeitet wird, ohne zu viel Zeit zu verlieren?
  - Kontakt-Webseite mit Feedback, Rudimentäre Kontaktwebseite und Feedback auslagern?
  - Kennst du vielleicht gute Ressourcen zum Lernen von D3?

### Backend

- Update von Maximo
- Update von Jan

## Ausblick
- AuDojo soll mehr als nur ein Teamprojekt sein
- Wollen Nachhaltig und vernünftig nach best practices programmieren -> skillcap schnell erreicht
- Können nicht all unsere Ziele und Anforderungen im Rahmen dieses Teamprojekts realisieren
 --> Möchten das Projekt gerne über den Umfang des Teamprojekts hinaus fortsetzen und erweitern (Clean Code, Gutes UI/Design , User Managment, AuD2, Theoretische Informatik, Logik...)
- Fragen:
  - Früheres hosten der Website möglich? (wenn SortSensei 1.0 fertig)
  - Bekommen wir eine Mail-Adresse für User-Feedback Bug-Reports gestellt?

**Comment by anomatomato** [comment]
Stark :muscle:
